lib/libzutil/zutil_import.c

```diff
@@ -1079,7 +1077,7 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 			error = zpool_read_label_slow(fd, config, num_labels);
 			saved_errno = errno;
 		}
-		free(labels);
```

Comment:
This free should become umem_free_aligned, not be deleted. Deleting it is a memory leak.

Reply:
I literally looked at this earlier and wondered why I left a blank line it.
Will do.